src/constants/constants.js

```diff
@@ -1 +1,2 @@
-export const API_URL = 'https://bootcamp-api.codeit.kr/api/sample'
+export const SHARED_PAGE_API_URL = 'https://bootcamp-api.codeit.kr/api/sample';
+export const FOLDER_PAGE_API_URL = 'https://bootcamp-api.codeit.kr/api/users';
```

Comment:
- 페이지, UI컴포넌트 이름 등은 이런 변수 네이밍에는 쓰지 않는게 좋습니다. 사용되는 곳이 UI일지라도 내용 자체만으로는 UI와 무관하고 무관하게 쓸 수 있으니까요. 그야말로 인터페이스기 때문에 바뀌기 쉬운(쉬워야하는) 부분입니다. 

- api 엔드포인트와 이름을 맞춰주는게 좋겠습니다

```suggestion
export const SAMPLE_API_URL = 'https://bootcamp-api.codeit.kr/api/sample';
export const USER_API_URL = 'https://bootcamp-api.codeit.kr/api/users';
```